frontends/bmd/test/test_utils.tsx

```diff
@@ -55,6 +57,7 @@ export function render(
     forceSaveVote = jest.fn(),
     userSettings = { textSize: GLOBALS.TEXT_SIZE as TextSizeSetting },
     votes = {},
+    random = mersenne(0),
```

Comment:
Creates a seeded generator in tests that always generates the same values. So for example, every ballot ID generated in test will be the same.